fixture-monkey-jakarta-validation/src/main/java/com/navercorp/fixturemonkey/jakarta/validation/generator/JakartaValidationNullInjectGenerator.java

```diff
@@ -73,8 +75,45 @@ public double generate(ObjectPropertyGeneratorContext context) {
 			}
 		}
 
-		if (context.isContainer() && annotations.contains(NotEmpty.class)) {
-			return 0.0d;
+		if (context.isContainer()) {
+			if (annotations.contains(NotEmpty.class)) {
+				return 0.0d;
+			}
+
+			Size sizeAnnotation = context.getProperty().getAnnotation(Size.class).orElse(null);
+			if (sizeAnnotation != null && sizeAnnotation.min() > 0) {
+				return 0.0d;
+			}
```

Comment:
It should be checked https://github.com/naver/fixture-monkey/pull/1208/files#r2174507127